Firestore/core/test/firebase/firestore/auth/firebase_credentials_provider_test.mm

```diff
@@ -89,6 +89,33 @@
   credentials_provider.SetUserChangeListener(nullptr);
 }
 
+FIRApp* FakeAppExpectingForceRefreshToken(NSString* _Nullable uid,
```

Comment:
FYI- I suspect there's no equivalent of these tests on JS / Android and so you probably won't be able to port this test...  but I'm not too concerned since the logic is so simple.